src/AppHead.tsx

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
+
+interface AppHeadProps {
+  title: string;
+  /**
+   * Og titles are expected to be more concise,
+   * whereas titles in the <title> attribute
+   * should be SEO-friendly
+   *
+   * ogTitle: My Great Machine
```

Comment:
With this
```suggestion
   * @example
   * ogTitle: My Great Machine
```
it can render slightly better in the IDE.

**Before**
<img width="543" alt="Screenshot 2021-08-24 at 14 02 49" src="https://user-images.githubusercontent.com/9800850/130613117-09323fc7-f82c-440b-917a-2ed0a33fdfed.png">
**After**
<img width="602" alt="Screenshot 2021-08-24 at 14 02 34" src="https://user-images.githubusercontent.com/9800850/130613223-684c2587-276a-4a55-9310-110cbeacea36.png">